libs/blocks/aside/aside.js

```diff
@@ -105,11 +105,61 @@ function decorateBlockBg(block, node) {
   }
 }
 
+function addPromobar(sourceEl, parent) {
+  const newPromo = sourceEl.cloneNode(true);
+  parent.appendChild(newPromo);
+}
+
+function checkViewportPromobar(foreground) {
+  const childCount = foreground.childElementCount;
+  const { children } = foreground;
```

Comment:
nit
```suggestion
  const { children, childElementCount: childCount } = foreground;
```

Reply:
@chrischrischris I have made the suggested changes.